packages/account/src/Helpers/utils.ts

```diff
@@ -183,9 +54,10 @@ export const shouldShowIdentityInformation = ({
 };
 
 export const getDocumentData = (country_code: string, document_type: string) => {
+    const IDV_DOCUMENT_DATA = getIDVDocumentConfig();
     return (
-        (Object.keys(idv_document_data).includes(country_code) &&
-            (idv_document_data as any)[country_code][document_type]) || {
+        (Object.keys(IDV_DOCUMENT_DATA).includes(country_code) &&
+            (IDV_DOCUMENT_DATA as any)[country_code][document_type]) || {
```

Comment:
@likhith-deriv is it possible to pass the `country_code` to `getIDVDocumentConfig()` function and only return the examples/config related to that single country? 
cc: @shahzaib-deriv